app/src/app/create/navtree/navtree.component.sass

```diff
@@ -12,16 +12,20 @@
     transition-property: background-color, color, border-color
     transition-duration: 500ms
     .index:after
-      content: ". "
+      content: " "
   .current-step
     background-color: $userbuttonbgcolor
     transition: background-color 300ms
   .jumpable-step
     cursor: pointer
+    span.stepName
+      display: none
   .disabled-step
     border-color: $disabled-frame-color
     color: $disabled-fg-color
     background-color: $disabled-bg-color
+    span.stepName
+      display: none
```

Comment:
PTAL
Here are two changements :

1. I add ". " in the names in create.service.ts, so that the jumpable step and the disabled step can only show the index number without ".". I think it's more beautiful.
2. The step's name will be hidden only for the screen width less than 414px （normal width for telephone）